gen/templates.go

```diff
@@ -229,7 +230,7 @@ var (
 // vendoredTemplates parses and returns vendored code generation templates.
 func vendoredTemplates() *template.Template {
 	_templates.Do(func() {
-		tmpl := template.New("templates").Funcs(templateFunctions())
+		tmpl := template.New("templates").Funcs(sprig.TxtFuncMap()).Funcs(templateFunctions())
```

Comment:
Only one function from `sprig` is used. I don't think it is worth adding a dependency.